llvm/lib/Passes/PassBuilderPipelines.cpp

```diff
@@ -1196,6 +1196,7 @@ PassBuilder::buildModuleSimplificationPipeline(OptimizationLevel Level,
     // In pre-link, we just want the instrumented IR. We use the contextual
     // profile in the post-thinlink phase.
     // The instrumentation will be removed in post-thinlink after IPO.
+    MPM.addPass(AssignUniqueIDPass());
```

Comment:
Should there be a TODO here to move this AssignUniqueIDPass earlier in the pipeline in the future?

Reply:
done. Come to think of it, it may eventually make sense for this to be an analysis, but this is for a RFC to decide. Left the